src/test/suite/fixtures/recorded/languages/rust/chuckNameDrum.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
```

Comment:
Not sure we need a "chuck" test here if you're not removing the `:` or doing anything interesting